mapbox_maps_flutter_mobile/example/integration_test/animation_test.dart

```diff
@@ -3,6 +3,9 @@ import 'dart:io';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 import 'package:mapbox_maps_flutter/mapbox_maps_flutter.dart';
+import 'package:mapbox_maps_flutter_interface/mapbox_maps_flutter_interface.dart'
```

Comment:
Not sure if this is the way to go - the examples need to import the public package not the internal one.